tokio/src/runtime/blocking/pool.rs

```diff
@@ -84,6 +84,19 @@ where
     rt.spawn_blocking(func)
 }
 
+cfg_fs! {
+    #[allow(dead_code)]
+    /// Runs the provided function on an executor dedicated to blocking operations.
+    pub(crate) fn spawn_mandatory_blocking<F, R>(func: F) -> JoinHandle<R>
```

Comment:
Is this dead code?

Reply:
It's dead in the (non-loom) build of the tests because a `spawn_mandatory_blocking` mock is used in `File`. 

I could write a test that uses `blocking::spawn_mandatory_blocking` so as to 

1) have a non-loom breathing test of the functionality 
2) avoid the dead code annotation

But I wanted to see the approach was good before putting time into this kind of thing

Reply:
Alright. We usually put conditions on `allow(dead_code)`, e.g.
```suggestion
cfg_fs! {
    #[cfg_attr(test, allow(dead_code))]
    /// Runs the provided function on an executor dedicated to blocking operations.
    pub(crate) fn spawn_mandatory_blocking<F, R>(func: F) -> JoinHandle<R>
```